lr_face/models.py

```diff
@@ -34,6 +36,27 @@ def __init__(self):
         super().__init__([Input(shape=(100, 100, 3)), Flatten(), Dense(100)])
 
 
+class FaceRecognition(tf.keras.Sequential):
+    """
+    A Face Recognition model that takes RGB images with dimensions as
+    input and outputs random embeddings with dimensionality 128. This keras
+    model won't be used, but the functions will be called directly from the
+    face_recognition library. Resolution (100, 100) is not used.'
+    """
+
+    def __init__(self):
+        super().__init__([Input(shape=(100, 100, 3)), Flatten(), Dense(128)])
```

Comment:
Why does this class inherit from `tf.keras.Sequential`? You don't use these layers for anything right?